main.py

```diff
@@ -1,9 +1,14 @@
 from adventurelib import *
 from protagonist import Protagonist
+
+from adv_c import adventure_c
```

Comment:
you need to import your starting function that calls start() then begins your game